entrypoint.sh

```diff
@@ -28,7 +28,8 @@ elif [ "$1" = "lint" ]; then
    # Install standard linting dependencies; YMMV
    pip --quiet install \
        .[lint] flake8 flake8-print flake8-logging-format flake8-isort
-   exec flake8 ${NAME}
+   flake8 ${NAME}
+    exit
```

Comment:
What does this do? Also seems to be misaligned?

Reply:
Not sure it's needed TBH as we're in an `if` statement and the script terminates afterwards.

Reply:
```suggestion
```

Reply:
I don't see any need for exit here but la me ask @marcglobality 

@marcglobality 
This change was introduced by you in this PR : https://github.com/globality-corp/globality-build/pull/480/files
Any comment?

Reply:
Created https://github.com/globality-corp/globality-build/pull/483

Reply:
merged